core/controllers/feedback.py

```diff
@@ -70,7 +76,7 @@ def post(self, exploration_id, thread_id):
                 'Text for the message must be specified.')
 
         feedback_services.create_message(
-            thread_id,
+            '.'.join((exploration_id, thread_id)),
```

Comment:
Oops, sorry, I don't mean generate_new_thread_id(). I mean some kind of method that takes an exp id and a thread id and concatenates them with a dot to get a 'full thread id' -- if it doesn't already exist it would be good to make one.